lib/Parser/Parse.cpp

```diff
@@ -12915,6 +12915,12 @@ void PrintPnodeWIndent(ParseNode *pnode,int indentAmt) {
       PrintFormalsWIndent(pnode->sxFnc.pnodeParams, indentAmt + INDENT_SIZE);
       PrintPnodeWIndent(pnode->sxFnc.pnodeRest, indentAmt + INDENT_SIZE);
       PrintPnodeWIndent(pnode->sxFnc.pnodeBody, indentAmt + INDENT_SIZE);
+      if (pnode->sxFnc.pnodeBody == nullptr)
+      {
+          Output::Print(_u("[%4d, %4d): "), pnode->ichMin, pnode->ichLim);
```

Comment:
Originally I thought that too but it turns out this is the prefix for each line of output in the AST trace.  Since I was printing a new line it looked more consistent having the prefix there, and also made it easier to do proper alignment of the text.

---
In reply to: [62569197](https://github.com/Microsoft/ChakraCore/pull/936#discussion_r62569197) [](ancestors = 62569197)